examples/03_evaluate/evaluation.ipynb

```diff
@@ -328,7 +328,7 @@
                     "name": "stderr",
                     "output_type": "stream",
                     "text": [
-                        "\r",
+                        "\r\n",
```

Comment:
when you submit the fixed version, could you please run the whole notebook so people can see the results?

Please use python 3.11 which is the latest one we support.